### DISCUSSION TITLE
Push notification OnEntry with IOS not always firing

### DISCUSSION THREAD

**Discussion by jjelecondev** [shinyorg/shiny#1520] [Q&A]
I am sure that I'm missing something.  I am using push notifications for both IOS and android.  I am not using  onReceived or anything meant to handle background.

On Android, notifications successfully do what they are supposed to do: when the notification item is tapped, OnEntry fires and my handling logic fires, redirecting to the correct in-app context.

On IOS -If the app is swiped away (i.e. closed), OnEntry fires and everything works correctly just as in the Android case

Here is the issue-  If the app is memory-resident but not in the foreground, tapping the push notification will switch to the app but OnEntry doesn't fire.

Again- sure I'm missing something so any insight will help.  Thanks in advance.

**Comment by aritchie** [comment]
This is not where issues go.  Also make sure to check the docs.  There is an FAQ section that may answer your question
